src/core/api/index.js

```diff
@@ -55,7 +55,7 @@ export function callApi({ endpoint, schema, params = {}, auth = false, state = {
     .then((response) => (schema ? normalize(response, schema) : response));
 }
 
-export function search({ api, page, query }) {
+export function adminSearch({ api, page, query }) {
```

Comment:
It doesn't change the endpoint. The admin URLs are all prefixed with `api/v3/internal` and the regular ones are `api/v3`. I moved the prefix into the config so it's the same now.